amgut/db/patches/0041.sql

```diff
@@ -841,18 +844,19 @@ INSERT INTO pm.sample (sample_id, is_blank, details)
 
 -- Add a plate
 INSERT INTO pm.plate_reader (name, notes)
-    VALUES ('PR1234', 'Standard plate reader');
+    VALUES ('Synergy HT', 'Knight Lab Plate reader');
 
 -- Add a mosquito machine
 INSERT INTO pm.mosquito (name, notes)
-    VALUES ('Mosquito1', 'Standard mosquito machine');
+    VALUES ('Mosquito left', 'Mosquito machine on the left'),
```

Comment:
These are the actual names from the wetlab? They'll need to retain those names...